etc/docker/Dockerfile.nvcr-triton-containers

```diff
@@ -0,0 +1,78 @@
+ARG TRITON_VERSION=24.06
+ARG JETPACK
+
+FROM nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-py3${JETPACK:+-igpu}-sdk AS nvidia-triton-build-base
+FROM nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-py3${JETPACK:+-igpu} AS nvidia-triton-runtime-base
+
+FROM nvidia-triton-build-base AS build-deps
+ENV DEBIAN_FRONTEND=noninteractive
+ENV HOME=/root
+
+# Create directories we will use
+RUN mkdir -p ${HOME}/opt/src
+
+RUN apt-get -y update && apt-get -y --no-install-recommends install \
+    ca-certificates \
+    gnupg
+
+RUN apt-key adv --fetch-key https://apt.kitware.com/keys/kitware-archive-latest.asc
+RUN echo 'deb https://apt.kitware.com/ubuntu/ jammy main' > /etc/apt/sources.list.d/kitware.list
+
+RUN apt-get -y update && apt-get -y --no-install-recommends install \
+    cmake \
+    git \
+    libabsl-dev \
+    libboost-all-dev \
+    libgrpc++-dev \
+    libprotobuf-dev \
+    ninja-build \
+    protobuf-compiler \
+    protobuf-compiler-grpc \
+    rapidjson-dev
+
+# Clone, build, and install the viam-cpp-sdk repo
+RUN mkdir -p ${HOME}/opt/src
+RUN cd ${HOME}/opt/src && \
+    git clone https://github.com/viamrobotics/viam-cpp-sdk.git -b releases/v0.0.9 && \
+    cd viam-cpp-sdk && \
+    cmake -S . -B build -G Ninja -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_PREFIX_PATH=/opt/viam && \
+    PATH=/opt/viam/bin:$PATH cmake --build build --target install -j3 -- -v && \
+    cmake --install build --prefix /opt/viam
+
+RUN mkdir -p /opt/tritonserver/include
+RUN ln -s /workspace/install/include/triton /opt/tritonserver/include/triton
+RUN ln -s /workspace/install/lib/stubs /opt/tritonserver/lib
```

Comment:
This unfortunate change is to work around the issues identified in the second bullet point of https://github.com/triton-inference-server/server/issues/6221#issuecomment-1758532314. Effectively, we fake an `/opt/tritonserver` directory that has only the triton specific pieces and hides their half-baked installations of protobuf and grpc that we can't use.